src/Microsoft.ML.StandardLearners/Standard/Online/OnlineLinear.cs

```diff
@@ -16,30 +16,49 @@
 
 namespace Microsoft.ML.Trainers.Online
 {
-
+    /// <summary>
+    /// Arguments class for online linear trainers.
+    /// </summary>
     public abstract class OnlineLinearArguments : LearnerInputBaseWithLabel
     {
-        [Argument(ArgumentType.AtMostOnce, HelpText = "Number of iterations", ShortName = "iter", SortOrder = 50)]
+        /// <summary>
+        /// Number of passes through the training dataset.
+        /// </summary>
+        [Argument(ArgumentType.AtMostOnce, HelpText = "Number of iterations", ShortName = "iter, numIterations", SortOrder = 50)]
```

Comment:
> numIterations", [](start = 96, length = 16)

Add a Name attribute property, for backwards compatibility, here and below in InitialWeightsDiameter. 

See: [https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs#L359](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs#L359)

Reply:
ShortName and Name work same way, so you can have it in either place.
On other hand ShortName assumes what it's a short name, which numIteration is not so short.
Also I'm not sure if you presence of space would make system happy `iter, numIterations` ->`iter,numIterations`

Reply:
i'll remove the space. numIterations is now "short", given the new name NumberOfIterations :)

---
In reply to: [257009222](https://github.com/dotnet/machinelearning/pull/2517#discussion_r257009222) [](ancestors = 257009222)